frontend/lib/verification/view/negative_verification_result.dart

```diff
@@ -10,37 +10,12 @@ class NegativeVerificationResult extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ContentCard(
-      borderSide: BorderSide(color: Colors.red, width: 4.0),
+    return VerificationResultCard(
+      title: "Die Ehrenamtskarte konnte nicht validiert werden!",
+      isPositive: false,
       child: Column(children: [
-        RichText(
-            textAlign: TextAlign.center,
-            text: TextSpan(
-                style: DefaultTextStyle.of(context)
-                    .style
-                    .apply(fontSizeFactor: 1.3),
-                children: [
-                  WidgetSpan(
-                    child: Padding(
-                      padding: const EdgeInsets.symmetric(horizontal: 2.0),
-                      child: Icon(
-                        Icons.error,
-                        color: Colors.red,
-                        size: 30,
-                      ),
-                    ),
-                  ),
-                  TextSpan(
-                      text: "Die Ehrenamtskarte konnte nicht"
-                          " validiert werden!\n\n",
-                      style: DefaultTextStyle.of(context)
-                          .style
-                          .apply(fontSizeFactor: 1.5, fontWeightDelta: 2)),
-                  verificationError.errorTextSpan,
-                  TextSpan(
-                      text: "\nFehlercode: ${verificationError.errorCode}"),
-                ])),
-      ]),
-    );
+        Text(verificationError.errorText),
+        Text("Fehlercode: ${verificationError.errorCode}"),
+      ]));
```

Comment:
Font gets really small with just Text(...), hardly readable on ios

Reply:
Oh, okay, I already wondered why the font is set to be so big 😆

Reply:
I did a UI rework in #279 so I would keep it as is in this PR, if you are okay with that.